com.rq3.Reaction.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: com.rq3.Reaction
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: Reaction
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --persist=.Reaction
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/SDL/sdl12-compat.json
+
+  - name: reaction
+    buildsystem: simple
+    build-commands:
+      - make DEFAULT_BASEDIR=${FLATPAK_DEST}/share/Reaction -j${FLATPAK_BUILDER_N_JOBS}
+      - mkdir -p ${FLATPAK_DEST}/bin/
+      - mkdir -p ${FLATPAK_DEST}/share/Reaction/
+      - make copyfiles COPYBINDIR=${FLATPAK_DEST}/bin/ COPYDIR=${FLATPAK_DEST}/share/Reaction/ NO_STRIP=1
+      - mv ${FLATPAK_DEST}/bin/Reactionded.* ${FLATPAK_DEST}/bin/Reactionded
+      - mv ${FLATPAK_DEST}/bin/Reaction.* ${FLATPAK_DEST}/bin/Reaction
+      - mv ${FLATPAK_DEST}/bin/*.so ${FLATPAK_DEST}/share/Reaction/
+      - install -Dm644 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 reaction.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: git
+        url: https://github.com/ReactionQuake3/reaction.git
+        tag: d214682124851c5fb4a1f7f254305ccfcec7bffc
+      - type: patch
+        path: resync.patch
+      - type: patch
+        path: freedesktop.patch
+        use-git: true
+
+  - name: boomstick
+    buildsystem: simple
+    build-commands:
+      - rm -f *.x86_64 *.so
+      - cp -ar Boomstick ${FLATPAK_DEST}/share/Reaction
+    sources:
+      - type: archive
+        url: https://download.rq3.com/Reaction-1.0-Linux-x86_64.tar.gz
```

Comment:
how does that work on aarch64?

Reply:
Notice that I just delete all platform specific pre-built binaries. I am only interested in the mod files, which are only included in these compiled forms together with an outdated engine. I could also pick the i386 download.